.github/dependabot.yml

```diff
@@ -1,28 +1,33 @@
 version: 2
 updates:
-  - package-ecosystem: 'npm'
-    directory: '/'
+  - package-ecosystem: "npm"
+    directories:
+      - "/"
+      - "/browser-tests"
     schedule:
       interval: daily
       time: "03:00"
     target-branch: main
     labels:
     - dependabot
     ignore:
-      - dependency-name: "node"
-        versions: ["17.x","18.x","20.x"]
       - dependency-name: "*"
         update-types: [ "version-update:semver-patch" ]
+      - dependency-name: "@govuk-one-login/data-vocab" # ignore until data-vocab is published to public npm registry, or we add token here
+    open-pull-requests-limit: 10
     commit-message:
       prefix: BAU
-  - package-ecosystem: docker
+  - package-ecosystem: "docker"
     directory: "/"
     schedule:
       interval: daily
       time: "03:00"
     target-branch: main
     labels:
     - dependabot
+    ignore:
+      - dependency-name: "node"
+        versions: [ ">=22.x" ]
```

Comment:
Do we need to explicitly ignore major updates indefinitely, as we can disable dependabot major updates with "@dependabot ignore this major version"

Reply:
Good point I guess we don't really no, I'll remove this